README.md

```diff
@@ -29,6 +29,8 @@ transXpress requires:
 * sra-tools (install via conda)
 * tidyverse (required for Trinity, install via conda)
 * python 3.6, numpy 1.16, scipy 1.0, theano 1.0.1, six 1.11 (required for deeploc, install via conda)
+* yaml (https://github.com/yaml/pyyaml)
+* ratelimiter (install via pip)
```

Comment:
We never needed to install yaml or ratelimiter separately, why is that required?

Reply:
I came across this issue when using a system-wide `python` and `snakemake`. I had to uninstall the `snakemake` to get any further but nevertheless provided there are two yaml implementations in python available I thought it is better to be explicit than implicit.

```
$ ./run_test.sh
Running the transXpress-trinity pipeline using snakemake
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.7/snakemake", line 33, in <module>
    sys.exit(load_entry_point('snakemake==5.32.2', 'console_scripts', 'snakemake')())
  File "/usr/lib/python3.7/site-packages/snakemake/__init__.py", line 2312, in main
    config = parse_config(args)
  File "/usr/lib/python3.7/site-packages/snakemake/__init__.py", line 889, in parse_config
    import yaml
ModuleNotFoundError: No module named 'yaml'
Making DAG file describing pipeline execution
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.7/snakemake", line 33, in <module>
    sys.exit(load_entry_point('snakemake==5.32.2', 'console_scripts', 'snakemake')())
  File "/usr/lib/python3.7/site-packages/snakemake/__init__.py", line 2312, in main
    config = parse_config(args)
  File "/usr/lib/python3.7/site-packages/snakemake/__init__.py", line 889, in parse_config
    import yaml
ModuleNotFoundError: No module named 'yaml'
```

Reply:
Regarding the `ratelimiter`:

```
$ ./run_test.sh
Running the transXpress-trinity pipeline using snakemake
DistributionNotFound in line 10 of /home/mmokrejs/proj/transXpress-snakemake/Snakefile:
The 'ratelimiter' distribution was not found and is required by snakemake
  File "/home/mmokrejs/proj/transXpress-snakemake/Snakefile", line 10, in <module>
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3239, in <module>
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3222, in _call_aside
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3251, in _initialize_master_working_set
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 567, in _build_master
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 884, in require
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 770, in resolve
Making DAG file describing pipeline execution
DistributionNotFound in line 10 of /home/mmokrejs/proj/transXpress-snakemake/Snakefile:
The 'ratelimiter' distribution was not found and is required by snakemake
  File "/home/mmokrejs/proj/transXpress-snakemake/Snakefile", line 10, in <module>
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3239, in <module>
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3222, in _call_aside
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3251, in _initialize_master_working_set
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 567, in _build_master
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 884, in require
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 770, in resolve
$
```
If one gets the deps installed through conda and pip then they do no show up, but if one tries to use a system-wide python, the list of dependencies is incomplete.

I am fine with dropping these two lines in the pull request.

Reply:
Did you create a conda environment for transXpress, as shown in the README?

Reply:
Yes, but the `/usr/bin/snakemake` was picked from my `PATH` and triggered using a system-wide installation of python to be used. As I wrote, I had to uninstall system-wide `snakemake`.

Reply:
conda should update your PATH as well. Maybe you just need to re-login so the new PATH is reflected.